packages/cfd/src/features/Containers/compare-accounts/compare-accounts.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import classNames from 'classnames';
+import { Text, Icon, PageOverlay, DesktopWrapper, MobileWrapper, CFDCompareAccountsCarousel } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import CFDCompareAccountsCard from '../../../Containers/cfd-compare-accounts/cfd-compare-accounts-card';
+import {
+    getSortedCFDAvailableAccounts,
+    getEUAvailableAccounts,
+    getMT5DemoData,
+    getDxtradeDemoData,
+    getCtraderDemoData,
+    dxtrade_data,
+    ctrader_data,
+} from '../../../Helpers/compare-accounts-config';
+import { REGION } from '../../../Helpers/cfd-config';
+
+const CompareAccounts = observer(() => {
+    const history = useHistory();
+    const store = useStore();
+    const { client, traders_hub } = store;
```

Comment:
Avoid using stores

Reply:
this will be handled in this PR: https://app.clickup.com/t/20696747/CFDS-1485